conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureBodySerDe.java

```diff
@@ -64,12 +66,12 @@
     }
 
     @Override
-    public <T> Serializer<T> serializer(TypeMarker<T> token) {
+    public <T> Serializer<T> serializer(TypeMarker<T> token, Optional<Endpoint> _endpoint) {
         return new EncodingSerializerRegistry<>(encodings, token);
```

Comment:
For the request-size-limit feature, we would have this registry wrap the encoding with a "request limiting wrapper"?

Reply:
Yep, via an encoding!